gcf/custom/manifest.go

```diff
@@ -145,20 +144,24 @@ func ComputeManifest(
 			}
 
 			if len(tableFolder.tmcf) > 0 && len(tableFolder.csv) > 0 {
+				// Only set automated_mcf_generation_by for tmcf/csv imports.
+				// Setting this flag for mcf based imports will error out.
+				manifestImport.AutomatedMcfGenerationBy = proto.String(importGroup)
```

Comment:
Note: this is from the following error message when automated_mcf_generation_by was specified in mcf only imports

import.has_resolution_info() && import.resolution_info().unresolved_mcf_urls_size() > 0 Import SEDAScores does NOT have correct resolution_info with unresolved_mcf_urls.